srcpkgs/iamb/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'iamb'
+pkgname=iamb
+version=0.0.9
+revision=1
+build_style=cargo
+configure_args="--no-default-features --features=native-tls"
+hostmakedepends="pkg-config"
+makedepends="oniguruma-devel sqlite-devel openssl-devel"
+short_desc="Matrix client for Vim addicts"
+maintainer="Bnyro <bnyro@tutanota.com>"
+license="Apache-2.0"
+homepage="https://iamb.chat/"
+distfiles="https://github.com/ulyssa/iamb/archive/refs/tags/v${version}.tar.gz"
+checksum=7ef6d23a957bfab62decd48caa83c106a49d95760b4b2ccf5a6b6a8f4506e687
+
+post_install() {
+	vinstall iamb.desktop 644 usr/share/applications
+	vinstall docs/iamb-256x256.png 644 usr/share/icons/hicolor/256x256/apps/iamb.png
```

Comment:
```suggestion
```

Reply:
Oh, I wanted to make one 512x512 but forgot to change it after copy and paste 😉